### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by KseniyaKuzmina**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,20 +3,34 @@
...
+![Resulting map](Images/resulting-image.png)
 
-<br/>
+To display information about a geographical point, do one of the following:
 
+* Locate a desired geographical point on the map and click it.
 
+* Specify coordinates of a geographical point in **Latitude** and **Longitude** text boxes and then click the **Send Geocode Request** button.
+
+After that the geographical point coordinates are passed to the [BingGeocodeDataProvider.RequestLocationInformation](https://docs.devexpress.com/WindowsForms/devexpress.xtramap.binggeocodedataprovider.requestlocationinformation.overloads) method. The response from the service contain an address ([LocationInformation.Address](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.LocationInformation.Address)), the entity type ([LocationInformation.EntityType](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.LocationInformation.EntityType)) and exact coordinates ([LocationInformation.Location](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.LocationInformation.Location)) that are shown in the **RichTextBox** element.
```
Comment (albertov05) [inline Readme.md:19]:
```suggestion
After that, the geographical point coordinates are passed to the [BingGeocodeDataProvider.RequestLocationInformation](https://docs.devexpress.com/WindowsForms/devexpress.xtramap.binggeocodedataprovider.requestlocationinformation.overloads) method. The response from the service contains an address ([LocationInformation.Address](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.LocationInformation.Address)), the entity type ([LocationInformation.EntityType](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.LocationInformation.EntityType)), and exact coordinates ([LocationInformation.Location](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.LocationInformation.Location)) that are shown in the **RichTextBox** element.
```

File: Readme.md [hunk 2]
```diff
@@ -3,20 +3,34 @@
...
 
+* Specify coordinates of a geographical point in **Latitude** and **Longitude** text boxes and then click the **Send Geocode Request** button.
+
+After that the geographical point coordinates are passed to the [BingGeocodeDataProvider.RequestLocationInformation](https://docs.devexpress.com/WindowsForms/devexpress.xtramap.binggeocodedataprovider.requestlocationinformation.overloads) method. The response from the service contain an address ([LocationInformation.Address](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.LocationInformation.Address)), the entity type ([LocationInformation.EntityType](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.LocationInformation.EntityType)) and exact coordinates ([LocationInformation.Location](https://docs.devexpress.com/WindowsForms/DevExpress.XtraMap.LocationInformation.Location)) that are shown in the **RichTextBox** element. 
+
+> **NOTE:** You need a key to use Bing Maps services in your application. Refer to the following topic for more information: [How to: Get a Bing Maps Key](https://docs.devexpress.com/WindowsForms/15102/controls-and-libraries/map-control/examples/general/how-to-get-a-bing-maps-key) tutorial.
```
Comment (albertov05) [inline Readme.md:21]:
```suggestion
> **NOTE:** You need a key to use Bing Maps services in your application. Refer to the following topic for more information: [How to: Get a Bing Maps Key](https://docs.devexpress.com/WindowsForms/15102/controls-and-libraries/map-control/examples/general/how-to-get-a-bing-maps-key).
```

File: Readme.md [hunk 3]
```diff
@@ -3,20 +3,34 @@
...
+* [Form1.cs](./CS/RequestLocationInformation/Form1.cs) (VB: [Form1.vb](./VB/RequestLocationInformation/Form1.vb))
+
+## Documentation
+
+* [Geocode](https://docs.devexpress.com/WindowsForms/16712/controls-and-libraries/map-control/gis-data/geocode)
+
+## More Examples
+
+* [How to Use the OpenStreetMap Geocode Service Via the Map Control](https://github.com/DevExpress-Examples/how-to-use-the-openstreetmap-geocode-service-via-the-map-control-t629683)
+* [How to get additional search results using the Bing Map Search service](https://github.com/DevExpress-Examples/how-to-get-additional-search-results-using-the-bing-map-search-service-e5066)
```
Comment (albertov05) [inline Readme.md:34]: Can you update this Readme.md title as follows?

`How to get additional search results using the Bing Maps Search service`

Comment (KseniyaKuzmina) [reply Readme.md]: Updated.

**Pull request merged**